### PR TITLE
XML documentation for DNN Image Featurizer Transfomer.

### PR DESCRIPTION
**Pull request by codemzs**
towards #3204 .

### CODE REVIEW

File: src/Microsoft.ML.OnnxTransformer/DnnImageFeaturizerTransform.cs [hunk 1]
```diff
@@ -41,11 +41,25 @@ internal DnnImageFeaturizerInput(string outputColumnName, string inputColumnName
     }
 
     /// <summary>
-    /// The Dnn Image Featurizer is just a wrapper around two <see cref="OnnxScoringEstimator"/>s and three <see cref="ColumnCopyingEstimator"/>
+    /// The Dnn Image Featurizer is just a wrapper around two <see cref="OnnxScoringEstimator"/> and three <see cref="ColumnCopyingEstimator"/>
```
Comment (sfilipi) [inline src/Microsoft.ML.OnnxTransformer/DnnImageFeaturizerTransform.cs]: let's remove 'just'. feels a diminutive.

File: src/Microsoft.ML.OnnxTransformer/OnnxCatalog.cs [hunk 2]
```diff
@@ -71,8 +71,8 @@ public static class OnnxCatalog
         => new OnnxScoringEstimator(CatalogUtils.GetEnvironment(catalog), outputColumnNames, inputColumnNames, modelFile, gpuDeviceId, fallbackToCpu);
 
         /// <summary>
-        /// Creates a new instance of <see cref="DnnImageFeaturizerEstimator"/> which applies one of the pre-trained DNN models in
-        /// <see cref="DnnImageModelSelector"/>to featurize an image.
+        /// Create <see cref="DnnImageFeaturizerEstimator"/>, which applies one of the pre-trained DNN models in
+        /// <see cref="DnnImageModelSelector"/> to featurize an image.
         /// </summary>
         /// <param name="catalog">The transform's catalog.</param>
         /// <param name="outputColumnName">The name of the column resulting from the transformation of <paramref name="inputColumnName"/>.</param>
```
Comment (sfilipi) [inline src/Microsoft.ML.OnnxTransformer/OnnxCatalog.cs]: >outputColumnName [](start = 25, length = 16)

type.

**Comment by sfilipi** [comment]
>         /// <param name="inputColumnName">Name of column to transform.

type. 

---
Refers to: src/Microsoft.ML.OnnxTransformer/OnnxCatalog.cs:82 in a17be90. [](commit_id = a17be90eb42f44e3fbec3ab1a75106aea13a7125, deletion_comment = False)

File: src/Microsoft.ML.OnnxTransformer/DnnImageFeaturizerTransform.cs [hunk 3]
```diff
@@ -41,11 +41,25 @@ internal DnnImageFeaturizerInput(string outputColumnName, string inputColumnName
...
     /// with present pretrained DNN models. The ColumnsCopying are there to allow arbitrary column input and output names, as by default
     /// the ONNXTransformer requires the names of the columns to be identical to the names of the ONNX model nodes.
     /// Note that because of this, it only works on Windows machines as that is a constraint of the OnnxTransformer.
```
Comment (sfilipi) [inline src/Microsoft.ML.OnnxTransformer/DnnImageFeaturizerTransform.cs]: >es  [](start = 66, length = 3)

comma, I believe.

File: src/Microsoft.ML.OnnxTransformer/DnnImageFeaturizerTransform.cs [hunk 4]
```diff
@@ -41,11 +41,25 @@ internal DnnImageFeaturizerInput(string outputColumnName, string inputColumnName
...
     /// </summary>
+    /// <remarks>
+    /// <format type="text/markdown"><![CDATA[
+    ///
+    /// ###  Estimator Characteristics
+    /// |  |  |
+    /// | -- | -- |
+    /// | Does this estimator need to look at the data to train its parameters? | No |
+    /// | Input column data type | Vector of <xref:System.Single> |
+    /// | Output column data type | <xref:System.Single> |
```
Comment (shmoradims) [inline src/Microsoft.ML.OnnxTransformer/DnnImageFeaturizerTransform.cs]: ><xref:System.Single> [](start = 36, length = 20)

Output is also a vector of single, i believe.

Comment (codemzs) [reply src/Microsoft.ML.OnnxTransformer/DnnImageFeaturizerTransform.cs]: Just verified with the sample, you are correct.

File: src/Microsoft.ML.OnnxTransformer/OnnxCatalog.cs (hunk 2 above)
Comment (shmoradims) [reply src/Microsoft.ML.OnnxTransformer/OnnxCatalog.cs]: add data type information


---
In reply to: [277200807](https://github.com/dotnet/machinelearning/pull/3456#discussion_r277200807) [](ancestors = 277200807)

File: src/Microsoft.ML.OnnxTransformer/DnnImageFeaturizerTransform.cs (hunk 3 above)
Comment (shmoradims) [reply src/Microsoft.ML.OnnxTransformer/DnnImageFeaturizerTransform.cs]: Windows machines, (add comma)

---
In reply to: [277200898](https://github.com/dotnet/machinelearning/pull/3456#discussion_r277200898) [](ancestors = 277200898)

**Pull request merged**